package.json

```diff
@@ -25,8 +25,8 @@
   "license": "MIT",
   "homepage": "https://github.com/carforyou/carforyou-components-pkg#readme",
   "peerDependencies": {
-    "react": ">=16.12.0",
-    "react-dom": ">=16.12.0",
+    "react": ">=17.0.0",
+    "react-dom": ">=17.0.0",
```

Comment:
Same as in header-footer, maybe we can update dev dependencies only?

Reply:
yes-ish. it gives me extra confidence that there would be no react16 laying around anywhere in other deps as it would halt the install. as we're on react17 already everywhere, I felt it's a safe move. I'm happy to adjust though. let me know what you think...

Reply:
Okay, I see. I'm not 100% sure it would halt the installation since react 17 allows multiple versions to co-exist IIRC.
I'm fine with going with it as is, no strong opinion.

Reply:
fair point, but our packages would now prevent us from having an implicit v16 somewhere. I'll move on with merging if your opinion is not too strong. I can always follow up if it causes issues